docs/_data/roadmap.yml

```diff
@@ -1,29 +1,34 @@
--
-  term: "Active development"
-  description: "Things actively being built right now."
-  items:
-    -
-      name: Client-side terrain rendering
-      issues: ["3730"]
-    -
-      name: Custom WebGL layer type
-      issues: ["281"]
-    - 
-      name: Property function support for <code>text-size</code> and <code>icon-size</code>
-      issues: ["4228"]
--
-  term: "Upcoming"
-  description: "Things we hope to start building in the next several months."
-  items:
-    -
-      name: Property aggregation on clustered features
-      issues: ["2412"]
-    -
-      name: <code>Map#setCamera</code> method
-      issues: ["3583", "2801"]
-    -
-      name: Custom source type API
-      project: 2
-    - 
-      name: Patterns for loading and errored tiles
-      issues: [3280]
+updated_at: '05/24/2017'
```

Comment:
This can be confusing for dates like 2/10/2017 is it 2nd of Oct or 10th of Feb? See https://en.wikipedia.org/wiki/Date_format_by_country#/media/File:Date_format_by_country_(new).png

I think it'll be safer to use something like 10th Feb 2017 or Feb 10th 2017 to avoid confusion.